### ISSUE TITLE
Undefined Method

### ISSUE THREAD

**Issue by stats** [cortiz/prawn-rails#18]
I am getting an error

``` ruby
 NoMethodError - undefined method `merge' for nil:NilClass:
  prawn-rails (0.1.1) lib/prawn-rails/document.rb:10:in `initialize'
  prawn-rails (0.1.1) lib/prawn-rails/rails_helper.rb:11:in `new'
  prawn-rails (0.1.1) lib/prawn-rails/rails_helper.rb:11:in `prawn_document'`
```

I looked at document.rb and it seems that PrawnConfig is nil. I have the initializer setup, and that appears to be working ok.

My document is: invoice.pdf.prawn and has the contents of

``` ruby
prawn_document do |pdf|
  pdf.text "Hello World!"
end
```

my prawn-rails.rb file contains:

``` ruby
PrawnRails.config do |config|
  config.page_layout = :portrait
  config.page_sie = "A4"
  config.skip_page_creation = false
end
```

and after running bundle update I see

```
Using prawn 1.3.0
Using prawn-table 0.2.1
Using prawn-rails 0.1.1
```

**Comment by stats** [comment]
It appears that to_h is not a valid method in ruby 1.9.3. It looks like to_h was added somewhere around ruby 2.0

I was able to replace to_h with marshal_dump on line 10 of document.rb to get it to work.

Either document.rb needs to be modified, or the Readme needs to be updated to show a Ruby 2.0 dependency.

**Comment by stats** [comment]
I have created a pull request which should fix this issue: https://github.com/cortiz/prawn-rails/pull/19

**Comment by mathieujobin** [comment]
should this issue be closed, since #19 got merged?
